pkgs/servers/radicale/default.nix

```diff
@@ -9,13 +9,7 @@ pythonPackages.buildPythonApplication rec {
     sha256 = "1c5lv8qca21mndkx350wxv34qypqh6gb4rhzms4anr642clq3jg2";
   };
 
-  propagatedBuildInputs = [
-    pythonPackages.flup
-    pythonPackages.ldap
-    pythonPackages.sqlalchemy
-  ];
```

Comment:
You probably should keep them for python2 in order not to break existing configs